tests/acceptance/helpers/backendHelper.js

```diff
@@ -0,0 +1,54 @@
+const { client } = require('nightwatch-api')
```

Comment:
`helpers/backendHelper` is redundant imo.

Reply:
why?

Reply:
it's already inside helpers, isn't it?